02-front-end/11bloco-react-intro/01dia-intro/exercicio-11-1/src/Footer.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class Footer extends Component {
+  render() {
+    return (
+      <div className='footer' >
+        <h1>E isso é só o começo...</h1>
+      </div>
```

Comment:
Não seria melhor utilizar uma tag mais semântica pra esse footer? 🤔
```suggestion
      <footer className='footer' >
        <h1>E isso é só o começo...</h1>
      </footer>
```

Reply:
Ótima ideia, nem pensei nisso na hora!